src/parser/bison_parser.y

```diff
@@ -756,67 +759,137 @@ update_clause : IDENTIFIER '=' expr {
 /******************************
  * Select Statement
  ******************************/
-
-select_statement : opt_with_clause select_with_paren {
-  $$ = $2;
-  $$->withDescriptions = $1;
-}
-| opt_with_clause select_no_paren {
-  $$ = $2;
-  $$->withDescriptions = $1;
-}
-| opt_with_clause select_with_paren set_operator select_within_set_operation opt_order opt_limit {
-  $$ = $2;
-  if ($$->setOperations == nullptr) {
-    $$->setOperations = new std::vector<SetOperation*>();
-  }
-  $$->setOperations->push_back($3);
-  $$->setOperations->back()->nestedSelectStatement = $4;
-  $$->setOperations->back()->resultOrder = $5;
-  $$->setOperations->back()->resultLimit = $6;
-  $$->withDescriptions = $1;
-};
-
-select_within_set_operation : select_with_paren | select_within_set_operation_no_parentheses;
-
-select_within_set_operation_no_parentheses : select_clause { $$ = $1; }
-| select_clause set_operator select_within_set_operation {
-  $$ = $1;
-  if ($$->setOperations == nullptr) {
-    $$->setOperations = new std::vector<SetOperation*>();
-  }
-  $$->setOperations->push_back($2);
-  $$->setOperations->back()->nestedSelectStatement = $3;
-};
-
-select_with_paren : '(' select_no_paren ')' { $$ = $2; }
-| '(' select_with_paren ')' { $$ = $2; };
-
-select_no_paren : select_clause opt_order opt_limit opt_locking_clause {
-  $$ = $1;
-  $$->order = $2;
-
-  // Limit could have been set by TOP.
-  if ($3) {
-    delete $$->limit;
-    $$->limit = $3;
-  }
-
-  if ($4) {
-    $$->lockings = $4;
-  }
-}
-| select_clause set_operator select_within_set_operation opt_order opt_limit opt_locking_clause {
-  $$ = $1;
-  if ($$->setOperations == nullptr) {
-    $$->setOperations = new std::vector<SetOperation*>();
-  }
-  $$->setOperations->push_back($2);
-  $$->setOperations->back()->nestedSelectStatement = $3;
-  $$->setOperations->back()->resultOrder = $4;
-  $$->setOperations->back()->resultLimit = $5;
-  $$->lockings = $6;
-};
+select_statement:
```

Comment:
pick here: The indentation does not match the other code. Could ypu please fix that (e.g., by calling the `format.sh` script)?

Reply:
Done.

Reply:
Btw, the formatting script has a bug where it places a space after % in a `%prec` declaration (see last commit)